### PR TITLE
Export 2021-12-08

### PR DESCRIPTION
**Pull request by nishant2718**
v96 Strings.

### CODE REVIEW

File: templates/firefox-ios.xliff [hunk 1]
```diff
@@ -1319,6 +1344,102 @@
...
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInBackground" xml:space="preserve">
+        <source>Open Link in Background</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab while staying on the currently selected tab. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.OpenLinkInNewTab" xml:space="preserve">
+        <source>Open Link in New Tab</source>
+        <note>A label indicating the keyboard shortcut of opening a link in a new tab and switching to that tab at the same time. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.SavePageAs" xml:space="preserve">
+        <source>Save Page As...</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1376]: This should use the "proper" ellipsis, which is just one character: `…`

File: templates/firefox-ios.xliff [hunk 2]
```diff
@@ -1319,6 +1344,102 @@
...
+        <source>Bookmarks</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with Bookmarks. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Edit" xml:space="preserve">
+        <source>Edit</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do within a web page. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.File" xml:space="preserve">
+        <source>File</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take on, and within, a Tab. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1389]: Is the comment relevant to the string "File"?
For some reason, I can't relate it to the string

File: templates/firefox-ios.xliff [hunk 3]
```diff
@@ -1319,6 +1344,102 @@
...
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.History" xml:space="preserve">
+        <source>History</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with History. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Tools" xml:space="preserve">
+        <source>Tools</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do with locally saved items. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1397]: Same comment as above
i'm having a hard time understanding how "Tools" is related to the comment

File: templates/firefox-ios.xliff [hunk 4]
```diff
@@ -1319,6 +1344,102 @@
...
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.View" xml:space="preserve">
+        <source>View</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can do regarding the viewing experience of a webpage. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.Section.Window" xml:space="preserve">
+        <source>Window</source>
+        <note>A label indicating a grouping of related keyboard shortcuts describing actions a user can take when navigating between their availale set of tabs. This label is displayed inside the Discoverability overlay when a user presses the Command key.</note>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1405]: Same question as above
Perhaps these comments I flag need to be tweaked a bit so localizers understand the context better, and can better localize

File: templates/firefox-ios.xliff [hunk 5]
```diff
@@ -1319,6 +1344,102 @@
...
+      <trans-unit id="Keyboard.Shortcuts.ShowLastTab" xml:space="preserve">
+        <source>Show Last Tab</source>
+        <note>A label indicating the keyboard shortcut switch from your current tab to the last tab. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.ShowTabNumber" xml:space="preserve">
+        <source>Show Tab Number 1-9</source>
+        <note>A label indicating the keyboard shortcut of switching between the first nine tabs. This label is displayed in the Discoverability overlay when a user presses the Command key.</note>
+      </trans-unit>
+      <trans-unit id="Keyboard.Shortcuts.ZoomIn" xml:space="preserve">
+        <source>Zoom Nn</source>
```
Comment (Delphine) [inline templates/firefox-ios.xliff:1436]: nit: should be "Zoom In"

**Review by Delphine — CHANGES_REQUESTED** [review]
Just a few comments, thank you! :)

One general additional thing is that it would be good to update all the keyboard shortcuts comments so they mention that "Discoverability overlay" is when a user connects a hardware keyboard/bluetooth to an iOS device - so that it makes it extra clear to localizers

File: templates/firefox-ios.xliff (hunk 2 above)
Comment (nishant2718) [reply templates/firefox-ios.xliff:1389]: According to design, certain actions should be grouped. So although these don't really relate directly to the string, they'll still exist under that category. You can have a look at this Figma as well: 

https://www.figma.com/file/4Vlr0idCIGQE3r9I1FnkRn/Keyboard-Shortcuts?node-id=2%3A7

**Comment by nishant2718** [comment]
For strings that act as categories for certain actions, what's your opinion on how to best handle them?

**Comment by Delphine** [comment]
Thanks for the explanation around the keyboard shortcuts!
Do you think we could add a link to a screenshot of that figma file (since localizers can't access Figma), and add that link at the end of all the keyboard shortcut strings' comments? that would be helpful to give additional context to localizers.
So, what I would suggest is something like "A label indicating the keyboard shortcut of (...). This label is displayed in the Discoverability overlay (ie. when someone adds a keyboard to an iOS device) when a user presses the Command key. See (link to screenshot) for more details."
I know this is kind of long,  but localizers will have all the context needed to translate correctly this way. I'd also suggest to add that comment to all the keyboard shortcut strings, as sometimes various localizers work on the same project - and they wouldn't see the link if it's not part of the comment of the string they are translating

**Comment by Delphine** [comment]
Thanks so much for those updates :)
It's looking good now 🤞🏼

**Pull request merged**